region/maridia/outer/Main Street.json

```diff
@@ -3042,6 +3042,50 @@
       "resetsObstacles": ["R-Mode"],
       "flashSuitChecked": true
     },
+    {
+      "link": [3, 3],
+      "name": "R-Mode Spark Interrupt",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        "Gravity",
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"or": [
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 0}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 60}}
+              ]},
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 1}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 40}}
+              ]}
```

Comment:
Is this right? You can farm 3 fish + 2 crabs with gravity and a crouch jump. Tho realistically, you'll have gravity jump as well and that gives even more enemies

Reply:
20E with not-full-ammo is 86% with 3 Sciser + 3 Skultera and 96% with all Scisers.